e2etest/pkg/metrics/metrics.go

```diff
@@ -47,6 +60,13 @@ func ForPod(source, target *corev1.Pod, namespace string) ([]map[string]*dto.Met
 	return allMetrics, nil
 }
 
+type CheckType int
+
+const (
+	NotThere CheckType = iota
+	There
+)
```

Comment:
could there be other values here in the future? if not, wouldn't it be a bit clearer just using a boolean?

Reply:
makes sense